src/main/java/com/zendesk/maxwell/Maxwell.java

```diff
@@ -33,6 +32,28 @@ private void initFirstRun(Connection connection) throws SQLException, IOExceptio
 		this.context.setPosition(pos);
 	}
 
+	private void setupFilter(MaxwellFilter filter, MaxwellConfig config) {
+		if (!config.include_databases.isEmpty()) {
+			for ( String d : config.include_databases.split(",") )
```

Comment:
Is there value in supporting spaces in the string? 
`.split("\\s*,\\s*")`

e.g. `include_databases=one, two, three`